easybuild/easyconfigs/d/darshan-util/darshan-util-3.4.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'darshan-util'
+version = '3.4.2'
+
+homepage = 'https://www.mcs.anl.gov/research/projects/darshan/'
+description = """Darshan, a scalable HPC I/O characterization tool. Darshan is 
+designed to capture an accurate picture of application I/O behavior, including 
+properties such as patterns of access within files, with minimum overhead.  
+The name is taken from a Sanskrit word for 'sight' or 'vision'. darshan-util
+can analyze log files produced by darshan-runtime"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@stevenvdb `GCCcore/8.3.0` is an unsupported toolchain (see https://docs.easybuild.io/policies/toolchains)

You mentioned that this is due to `Pod::Select` not being present in more recent `Perl` easyconfigs, perhaps we should simply fix that, so this can be updated to for example `GCCcore/10.3.0` (which matches with the `*/2021a` toolchain used for `darshan-runtime`).
Another option is to add `Pod::Select` as a dependency on top of a recent `Perl`.

Or perhaps a more recent version of `darshan-utils` can be used (not sure if that would help).

Reply:
I have opened a new PR with a recent toolchain. `Pod::Select` is provided by `Perl-bundle-CPAN/5.36.1-GCCcore-12.3.0`, so I used that as a starting point. I propose to close this PR and continue in #19643